packages/ariakit/src/tooltip/__examples__/tooltip/test.tsx

```diff
@@ -0,0 +1,20 @@
+import { getByRole, hover, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render tooltip", () => {
+  render(<Example />);
+  expect(getByRole("button")).toMatchInlineSnapshot(`
```

Comment:
Here we should get the `baseElement` because the tooltip is rendered in a portal element.
```suggestion
  const { baseElement } = render(<Example />);
  expect(baseElement).toMatchInlineSnapshot(`
```